### PR TITLE
Integrate Styled components in About Pages #2124

### LINKED ISSUE #2124: Integrate Styled components in All About Pages
**Actual Behaviour**

Not present

**Expected Behaviour**

Integrate styled components in all components of about page. Create common UI components to be reused.

**Would you like to work on the issue?**

Yes

### PR DESCRIPTION
**Pull request by fragm3**
Fixes #2124

Changes:
1. Integrate Styled components in All About Pages
2. Remove redundant CSS
3. Fix mobile view for example section in Overview

Demo Link: https://pr-2125-1-fossasia-susi-web-chat.surge.sh

### CODE REVIEW

**Comment by fragm3** [comment]
@akshatnitd I am not sure if integrating styled components in these pages for all sections would be better.

File: src/components/Commons/Close.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import styled from 'styled-components';
+import CloseIcon from '@material-ui/icons/Close';
+
+const Close = styled(CloseIcon)`
```
Comment (akshatnitd) [inline src/components/Commons/Close.js]: Use material-ui's IconButton component here

File: src/components/Commons/About.js [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+const SubTitle = styled.p`
+  font-size: 1.4rem;
+  font-weight: 300;
+  line-height: 2.3rem;
+  margin: 1rem auto;
+  max-width: 32rem;
+  color: #414141;
+`;
+
+export const Header = ({ title, subtitle }) => (
```
Comment (akshatnitd) [inline src/components/Commons/About.js]: This component should be restructured. Take title, subtitle as props. and then render the children.
Eg : 
```
<div>
{props.title}
{props.subtitle}
{props.children}
</div>
```

Comment (akshatnitd) [reply src/components/Commons/About.js]: It would be more of a wrapper component. And it shouldn't be a shared component. It is a very specific use case for the About Section.

Comment (fragm3) [reply src/components/Commons/About.js]: @akshatnitd Where should this be placed? The About section components are not present in a single directory. We can either place the About section components together or place this wrapper component in Shared itself

**Comment by fragm3** [comment]
@akshatnitd made the changes

File: public/403.html [hunk 3]
```diff
@@ -13,7 +13,7 @@ <h3>Forbidden | The Browser you're currently using does not support <strong>susi
   </div>
   <div class="footer">
     <a href="https://chat.susi.ai/" title="Home" target="_blank">SUSI.AI<span></span></a>
-    <p class="legal">&copy;Copyright Creative Commons By License, FOSSASIA&trade;</p>
+    <p class="legal">&copy;Copyright Creative Shared By License, FOSSASIA&trade;</p>
```
Comment (akshatnitd) [inline public/403.html]: Why did you change this?

File: src/__tests__/components/Shared/ClosedButton.js [hunk 4]
```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+import CloseButton from '../../../components/Shared/CloseButton';
```
Comment (akshatnitd) [inline src/__tests__/components/Shared/ClosedButton.js]: The folder name should be `shared`, not `Shared`.

File: src/components/Shared/About.js [hunk 5]
```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
```
Comment (akshatnitd) [inline src/components/Shared/About.js]: It should be under Settings folder, and rename the component to `SectionWrapper` instead.

File: src/images/susi-logo.svg [hunk 6]
```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8" standalone="no"?>
 <svg
    xmlns:dc="http://purl.org/dc/elements/1.1/"
-   xmlns:cc="http://creativecommons.org/ns#"
+   xmlns:cc="http://creativeShared.org/ns#"
```
Comment (akshatnitd) [inline src/images/susi-logo.svg]: Even this change. Your find replace has gone crazy :D

File: src/components/Shared/About.js (hunk 5 above)
Comment (fragm3) [reply src/components/Shared/About.js]: But this has nothing to do with Settings folder, it is for the overview, blogs, etc static pages

**Comment by fragm3** [comment]
@akshatnitd I have renamed both test and component folder from Shared to share, but it's not reflected in the changes

**Comment by fragm3** [comment]
@akshatnitd made the changes, it was some Github bug, the folder name change was not getting recognized.
